### DISCUSSION TITLE
When can Repeater add the files field?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#4688] [Archive]
I once saw the description of this feature in an issue, but I did not find it on the Road Map of the official website, so I want to ask if there is a plan.

It is available in a certain version of v8, and I use it for almost every piece of data. It may become my biggest obstacle to migrating data to v9.

😋😋

**Comment by benhaynes** [comment]
Ref: https://github.com/directus/directus/discussions/3884

**Comment by benhaynes** [comment]
Merging this into: https://github.com/directus/directus/discussions/3001
